### PR TITLE
DeltaInt

### PR DESCRIPTION
**Pull request by ice1000**
Supersedes #168

### CODE REVIEW

**Comment by ice1000** [comment]
The idea is provided by @kaa1el

**Comment by Alizter** [comment]
Let's reset this now that #179 has been merged.

**Comment by ice1000** [comment]
CI passes!! @mortberg @Saizan  please take a look

File: Cubical/HITs/Ints/DeltaInt/Base.agda [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{-# OPTIONS --cubical --safe #-}
+
+{-
+
+This file defines DiffInt (Cubical.HITs.DiffInt) as HIT.
```
Comment (Saizan) [inline Cubical/HITs/Ints/DeltaInt/Base.agda]: I guess it should be `DeltaInt` in both those cases?

**Comment by Saizan** [comment]
Looks good, if you fix that typo I will merge.

File: Cubical/HITs/Ints/DeltaInt/Base.agda (hunk 1 above)
Comment (ice1000) [reply Cubical/HITs/Ints/DeltaInt/Base.agda]: Nope, I meant the following:

https://github.com/agda/cubical/blob/6ca71118f1275b4b5a6b4c4e37669b6d469fcbad/Cubical/Data/DiffInt.agda#L1-L5

But it's still a typo (`HITs` -> `Data`), I'll fix

**Comment by ice1000** [comment]
Changed `HITs` to `Data`

**Pull request merged**